test/e2e/util.go

```diff
@@ -1918,12 +1937,28 @@ func isNodeReadySetAsExpected(node *api.Node, wantReady bool) bool {
 	return false
 }
 
+func isNodeOutOfDiskSetAsExpected(node *api.Node, wantOutOfDisk bool) bool {
```

Comment:
I'm not one to usually insist on refactoring common code, but in this case I think it's warranted (this function and isNodeReadySetAsExpected() are identical except for which condition they check) -- just rename isNodeReadySetAsExpected() to isNodeConditionSetAsExpected() and have it take a cond.Type as an additional argument. Then you don't need to add a funciton.